builtin/common/misc_helpers.lua

```diff
@@ -205,6 +205,16 @@ function table.indexof(list, val)
 	return -1
 end
 
+--------------------------------------------------------------------------------
+function table.keyof(list, val)
```

Comment:
I wouldn't call the table `list` in this context. I'd go with something like `tab` or `tbl`.

Reply:
Done in e19d07549079eff0859ab882d2328891747cf5ae